src/settings.ts

```diff
@@ -154,7 +154,7 @@ export const PLAYA_VENUE_STYLES: Record<string, CSSProperties> = {
 };
 
 export const ACCEPTED_IMAGE_TYPES =
-  "image/png,image/x-png,image/gif,image/jpg,image/jpeg,image/tiff,image/bmp,image/gif";
+  "image/png,image/x-png,image/gif,image/jpg,image/jpeg,image/tiff,image/bmp,image/gif,image/webp";
```

Comment:
May be with submitting this commit as a separate PR right away against staging?

Reply:
Good call -- thanks! https://github.com/sparkletown/sparkle/pull/1601